backend/sendmessage/src/app.ts

```diff
@@ -1,58 +1,26 @@
 import * as AWS from 'aws-sdk';
 import { APIGatewayProxyHandler } from 'aws-lambda';
 import { Config } from './types';
-import { loginUser } from './login-user';
-import { resetVotes } from './reset-votes';
-import { removeUsersNotVoted } from './remove-users-not-voted';
-import { setScale } from './set-scale';
-import { revealVotes } from './reveal-votes';
-import { setVote } from './set-vote';
+import { message } from './message';
+import { TABLE_NAME } from './const';
 
 const ddb = new AWS.DynamoDB.DocumentClient({
   apiVersion: '2012-08-10',
   region: process.env.AWS_REGION,
 });
 
-const TABLE_NAME = process.env.TABLE_NAME || 'scrum_poker';
-
 export const handler: APIGatewayProxyHandler = async (event) => {
-  const apigwManagementApi = new AWS.ApiGatewayManagementApi({
-    apiVersion: '2018-11-29',
-    endpoint: event.requestContext.domainName,
-  });
   const connectionId = event.requestContext.connectionId;
   const { type, payload } = JSON.parse(event.body as string).data;
```

Comment:
If we do not destructure here but keep it as a "message" object, we might in the future share message types with the frontend. Currently, there are interfaces defined in `WebSocket.ts` for each message that connect the type with the shape of the payload.
While I would not do it necessarily in this PR, in the future we could e.g. share a `d.ts` file between frontend and backend containing these types to ensure interface compatibility (and replace the current `Payload` type).

(I think an ambient d.ts file should not cause issues with the current backend transpilation process).